projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/control/LLVMSwitchNode.java

```diff
@@ -130,6 +129,7 @@ public LLVMI8ProfilingSwitchNode(LLVMI8Node cond, LLVMI8Node[] cases, int[] succ
         }
 
         @Override
+        @ExplodeLoop
         boolean profile(int i, boolean value) {
             return profiles[i].profile(value);
         }
```

Comment:
Yes, I agree that it does not make sense. I am not sure how I came up with using the annotation here.